### PR TITLE
Add new field to thr ucr for new image question

### PR DESCRIPTION
**Pull request by Rohit25negi**
This is preparation work for new THR report coming:
Functional Spec: https://docs.google.com/document/d/1dAUPZoze-n7-rIpnmbdXuyPyler4_oCuWt4Ji7qO6zw/edit?ts=5cfb9fd0

Tech spec: https://docs.google.com/spreadsheets/d/1UIp1w2vzzckrKQMXsIcsaCDgifGhZ7nB/edit#gid=723565633

### CODE REVIEW

File: custom/icds_reports/ucr/tests/data/thr_form_v2.xml [hunk 1]
```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" ?>
```
Comment (Rohit25negi) [inline custom/icds_reports/ucr/tests/data/thr_form_v2.xml]: This xml is from India server. Total test data given by QA team. So nothing to worry about data leak.

File: custom/icds_reports/ucr/data_sources/thr_forms_v2.json [hunk 2]
```diff
@@ -121,6 +139,22 @@
...
+        "expression": {
+          "type": "property_path",
+          "property_path": [
+            "form",
+            "photo_thr_packets_distributed"
+          ]
+        },
+        "is_primary_key": false,
+        "is_nullable": true,
+        "column_id": "image_name",
```
Comment (emord) [inline custom/icds_reports/ucr/data_sources/thr_forms_v2.json]: I think this would be more obvious if named `photo_thr_packets_distributed`

Comment (Rohit25negi) [reply custom/icds_reports/ucr/data_sources/thr_forms_v2.json]: Good point!. Changed the name.

**Comment by Rohit25negi** [comment]
@calellowitz @emord does it look good to you?

**Review by emord — APPROVED** [review]

**Pull request merged**